dts/bindings/sensor/st,vl53l0x.yaml

```diff
@@ -12,3 +26,25 @@ properties:
     type: phandle-array
     description: |
       Xshutdown pin, Active LOW.
+
+  max-time-for-ranging:
+    type: int
+    default: 33000
+    description: |
+      Set max time for ranging in μs
```

Comment:
I think you need to explain the reason for default value.

Reply:
ok

Reply:
resolved

Reply:
The "explanation" just refers to a preprocessor macro defined in the driver. It doesn't actually explain why the value was selected. Is it a register reset value in the sensor?

Same comment applies to the additional new properties defined below.